wp-content/themes/core/assets/css/src/theme/media/wp-audio-video.pcss

```diff
@@ -18,3 +22,53 @@
 		background-image: none;
 	}
 }
+
+/* -----------------------------------------------------------------------------
```

Comment:
@ryanurban just pointing out these styles to support WordPress block editor oEmbeds while we suss out our approach for custom stuff.